source/game.c

```diff
@@ -52,6 +213,10 @@ static Sprite *blind_select_tokens[MAX_BLINDS] = {NULL}; // The sprites that dis
 static int current_blind = SMALL_BLIND;
 static enum BlindState blinds[MAX_BLINDS] = {BLIND_CURRENT, BLIND_UPCOMING, BLIND_UPCOMING}; // The current state of the blinds, this is used to determine what the game is doing at any given time
 
+static int blind_reward = 0;
+static int hand_reward = 0;
+static int interest_reward = 0;
```

Comment:
Definitely something for later is encapsulating things into structs. Not necessary for this PR, but something to think of going forward and taking opportunities to fix when possible.